testing/test_config.py

```diff
@@ -1058,8 +1058,12 @@ def test_override_ini_usage_error_bad_style(self, testdir):
             xdist_strict=False
         """
         )
-        result = testdir.runpytest("--override-ini", "xdist_strict True", "-s")
```

Comment:
Maybe keep this test also? Doesn't really hurt to test both cases.

Reply:
But it is also unnecessary - this just removes the confusion from the test.

It tested `pytest -o "xdist_strict True"` before, but was meant to test `pytest -o xdist_strict True` really.
In both cases there's no "=" in the arg, which is what needs to be tested.